automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/NumberField.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.firecloud.component
+
+import org.openqa.selenium.WebDriver
+
+case class NumberField(queryString: QueryString)(implicit webDriver: WebDriver) extends Component(queryString) {
```

Comment:
Added this because the Notebooks Page has number fields - it's kind of looks too much like TextField though - we should think more about this implementation

Reply:
I wonder if `setText` / `getText` should just take Strings. As it stands now it wouldn't work with floating points. Selenium `NumberField` takes Strings: http://doc.scalatest.org/3.0.1-2.12/org/scalatest/selenium/WebBrowser$NumberField.html